### ISSUE TITLE
IBC channels can be stopped

### ISSUE THREAD

**Issue by faddat** [cosmos/ibc-go#6606]
IMPORTANT: Prior to opening a bug report, check if it affects one of the core modules
and if its elegible for a bug bounty on `SECURITY.md`. Bugs that are not submitted
through the appropriate channels won't receive any bounty.

Heh, if you report through "proper" channels, you experience:

* Retaliation
* No bounty

So there's genuinely no sense in that. 

## Summary of Bug

Ibc channels can be stopped by overwhelming them with transactions.  Currently tested only on ibc 1-7 with fully compliant 725kb transactions. 

Test tooling is available at 

GitHub.com/somatic-labs/hardhat

Replication of this issue with that tooling can be accomplished in minutes. 

## Expected Behaviour

Teams build with IBC sometimes with the expectation that their chains are able to make time-sensitive transactions.

In most cases this is true, however when there are lots of big transactions, the channel ceases to be useful. 

## Version

1-7 have been tested but I suspect this is universal. 

## Steps to Reproduce

```
git clone https://GitHub.com/somatic-labs/hardhat
cd hardhat
go install ./...
cd configurations/mars
```

* Make a file named seedphrase in the mars folder and put a seedphrase in it
* Check the config file

```
hardhat
``|


____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged/assigned

**Comment by crodriguezvega** [comment]
Thank you for reporting this issue, @faddat. We will discuss it internally with the utmost urgency and will try to come back to you as soon as possible.

**Comment by faddat** [comment]
The interchain foundation is abusive to reporters of security issues and ignores serious issues to the detriment of the entire cosmos ecosystem.

**Comment by faddat** [comment]
Doesn't seem to happen with lots of smalls.  

Maybe.  

Not sure.  

Been working this mainly solo since Sept 21 2023 when I reported similar but different issues to the interchain foundations approved security channels. 

I had conversations with the following groups about this:

* Informal systems
* Interchain gmbh
* Hypha
* Amulet 
* Interchain foundation 
* Ethan Buchman directly

Here are my full documents provided to the interchain foundation at that time:

**Comment by faddat** [comment]
@DimitrisJim yeah I understand about the non-technical stuff, but I do want you to know that I am reporting those items as well. 

So you recognized that these oversized transactions can turn off any active IBC Channel?

...as that's a technical thing.

**Comment by faddat** [comment]
For reference, there are 180,000 packets on umee waiting to get to cosmoshub

**Comment by faddat** [comment]
![Uploading image.png…]()


... osmo to hub

**Comment by faddat** [comment]
I'm really trying to also take care of the really grave security issues in the reporting process in Cosmos. This was a total failure, 0 out of 10.

And I want to make really clear that it absolutely was not only Jack talking about me instead of technology, it was our illustrious former Foundation president Bucky as well 

This raises very serious questions about priorities in Cosmos and questions about the viability of Cosmos and the interchain Foundation
![signal-2024-06-17-16-45-24-509-7](https://github.com/cosmos/ibc-go/assets/7142025/e94225b1-0c36-4b6c-a8f4-629007bd1634)
![signal-2024-06-17-16-45-24-509-6](https://github.com/cosmos/ibc-go/assets/7142025/f7680427-41b3-478f-b6ff-7dc863d7f501)
![signal-2024-06-17-16-45-24-509-3](https://github.com/cosmos/ibc-go/assets/7142025/a5bc3cdd-da22-49ab-bce0-1021765c8229)
![signal-2024-06-17-16-45-24-509-4](https://github.com/cosmos/ibc-go/assets/7142025/8bd1239c-47f3-42a6-9103-adb253023d01)

**Comment by faddat** [comment]
I hope that this issue serves as a warning to people in Cosmos that you can bring up an issue in an accurate way, and see every ICF funded team discuss you, instead of the issue. 

Information that can be gleaned from a block explorer is not about any persons personal style.  

I don't know what layer of the stack this issue is at.

Icf funds went into discussing me, not problems. 

There's no place to report stuff like this, because well, when I reported this stuff,  the organizational priority of the interchain foundation execs was on quashing reports. 

The screenshots are from eight months ago.

**Comment by crodriguezvega** [comment]
We discussed yesterday and we decided to backport https://github.com/cosmos/ibc-go/pull/4992 to v7 as a mitigation for this issue, and we will release v7.6.0 this week. Thank you @faddat for raising this issue.

**Comment by crodriguezvega** [comment]
We have released [v7.6.0](https://github.com/cosmos/ibc-go/releases/tag/v7.6.0) with the back port of #4992.
